interface/forms/group_attendance/view.php

```diff
@@ -0,0 +1,109 @@
+<?php
+include_once("../../globals.php");
```

Comment:
this script looks the same as interface/forms/group_attendance/new.php . Check out other forms for examples. Note most of the new forms have the script in new.php and then view.php is a include:
https://github.com/openemr/openemr/blob/master/interface/forms/care_plan/view.php